62_CAD/DrawResourcesFiller.cpp

```diff
@@ -684,6 +787,33 @@ uint32_t DrawResourcesFiller::addLineStyle_Internal(const LineStyleInfo& lineSty
 	return currentLineStylesCount++;
 }
 
+uint32_t DrawResourcesFiller::addDTMSettings_Internal(const DTMSettingsInfo& dtmSettingsInfo, SIntendedSubmitInfo& intendedNextSubmit)
+{
+	DTMSettings dtmSettings;
+
+	// TODO: this needs to be redone.. what if submit happens after that line?
+	// we need to make sure somehow that function below will not submit, we need both outline and contour styles in GPU memory
+	dtmSettings.outlineLineStyleIdx = addLineStyle_SubmitIfNeeded(dtmSettingsInfo.outlineLineStyleInfo, intendedNextSubmit);
```

Comment:
Asset here that the max line styles is >= 2, because you need at least 2 resident lineStyles. And you don't want the second call to evict the first